From 4f07ffa8f5ab85011422bb7114f1cddf49d4923d Mon Sep 17 00:00:00 2001
From: ChenQi1989 <40684930+ChenQi1989@users.noreply.github.com>
Date: Fri, 29 Jun 2018 06:54:07 +0000
Subject: Use #if instead of #ifdef for ENABLE_GSHADOW

ENABLE_GSHADOW is defined to be 0 or 1. So #if should be used instead of #ifdef.

Upstream-Status: Backport [https://github.com/systemd/systemd/commit/4f07ffa8f5ab85011422bb7114f1cddf49d4923d]
---
 src/basic/user-util.h | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/src/basic/user-util.h b/src/basic/user-util.h
index b74f16885..b2f198c89 100644
--- a/src/basic/user-util.h
+++ b/src/basic/user-util.h
@@ -102,7 +102,7 @@ int fgetgrent_sane(FILE *stream, struct group **gr);
 int putpwent_sane(const struct passwd *pw, FILE *stream);
 int putspent_sane(const struct spwd *sp, FILE *stream);
 int putgrent_sane(const struct group *gr, FILE *stream);
-#ifdef ENABLE_GSHADOW
+#if ENABLE_GSHADOW
 int fgetsgent_sane(FILE *stream, struct sgrp **sg);
 int putsgent_sane(const struct sgrp *sg, FILE *stream);
 #endif
-- 
2.18.1

